crds/cluster-lifecycle-chart/cluster.open-cluster-management.io_clustercurators.yaml

```diff
@@ -0,0 +1,246 @@
+# Copyright Contributors to the Open Cluster Management project
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+  creationTimestamp: null
+  name: clustercurators.cluster.open-cluster-management.io
+spec:
+  group: cluster.open-cluster-management.io
+  names:
+    kind: ClusterCurator
+    listKind: ClusterCuratorList
+    plural: clustercurators
+    singular: clustercurator
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    schema:
+      openAPIV3Schema:
+        description: ClusterCurator is the Schema for the clustercurators API
```

Comment:
I think it might need  a brief description on the API.  What it is used for, how the field in spec is picked and what condition type will be reported. It will need a change on the original repo, so may consider it as a followup.

Reply:
Done.